sources/packages/api/src/services/student/student.service.ts

```diff
@@ -119,19 +128,20 @@ export class StudentService extends RecordDataModelService<Student> {
     externalEntityManager?: EntityManager,
     studentAccountApplicationId?: number,
   ): Promise<Student> {
+    // SIN to be saved and used for comparisons.
+    const studentSIN = removeWhiteSpaces(studentInfo.sinNumber);
+    const student =
+      (await this.getExistingStudentForAccountCreation(studentSIN, userInfo)) ??
```

Comment:
Please create a variable for "await this.getExistingStudentForAccountCreation(studentSIN, userInfo)" just for some better understanding.

Reply:
Chnaged.